src/main/java/com/conveyal/r5/profile/RaptorState.java

```diff
@@ -337,4 +340,28 @@ public boolean stopWasUpdated (int stop, boolean withinMinute) {
         }
     }
 
+    /**
+     * Checks whether boarding a at one stop allows for shorter access/transfer legs than boarding at another stop.
```

Comment:
The Javadoc should specify (maybe with `@param`) that here and there are stop indexes, and that a return value of `true` means here is better than there.

As this is an instance method in the context of a single round's state, we should also make it clear that we're talking about boarding in the _next_ round, not boarding in this round. But the meaning of the method might be clearer if we just don't mention boarding at all, only what the method computes. It simply tells whether the walk or access leg to reach one stop is shorter than the other within this round.